### ISSUE TITLE
accessibility #3 -The id should be unique when used in labels.  

### ISSUE THREAD

**Issue by yulgit1** [ycba-cia/blacklight-collections2#233]
The id should be unique when used in labels.  There is a dummy search field that is using the same id as the default search field (id=”search_field”).

**Comment by yulgit1** [comment]
resolved: https://github.com/ycba-cia/blacklight-collections2/commit/41a9c599ab75ee0a34721b4b3a5e34ebe7f042a3
